programmers/학교습.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Vujahday+Script&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="학교습.css" />
+    <script src="https://kit.fontawesome.com/4865ddbb32.js" crossorigin="anonymous"></script>
+    <title>한결오빠가 준 연습과제</title>
+  </head>
+  <body>
+<container class = "con">
+    <header><div class="img"></div>
+    <img src="C:\Users\미나\Desktop\js\study\연습\platter-g3093ddc78_1920.jpg" alt="" class="device" />
+      <div class="time">
+          
+        <!-- <div class="left">8:25</div><div class="right">
+        <i class="fas fa-wifi"></i>
+        <i class="fas fa-battery-three-quarters"></i></div>
+      </div>
+         <i class="fas fa-arrow-left"></i></div>-->
+         <h2 class="h1">Blackstarz<br />Coffee</h2v>
+     
+    </header> 
+    <nav>
+        <div class="navwrap">
+    <div class="a1">
+        <i class="far fa-calendar-alt"></i>
+      <div class="date">13 Jan</div>
+      <div class="verticalLin"></div>
+    </div>
+    <div class="a2">
+        <i class="far fa-clock"></i>
+      <div class="day-time">12:30</div>
+      
+    </div>
+    <div class="a3">
+        <i class="fas fa-user"></i>
+      <div class="people">-</div>
+      
+    </div>
+</div>
```

Comment:
```suggestion
            </div>
```
depth (탭) 의 깊이를 맞춰주세요. 저장 혹은 특정 단축키를 사용하여 자동 정렬이 가능한 Extension을 사용하면 편합니다. (ex :
![Screen Shot 2022-01-13 at 10 30 06 PM](https://user-images.githubusercontent.com/69745441/149339158-d43296cf-b216-4ef0-8814-28f9e6a18ea1.png))